tests/meta/rsync/test_rsync.py

```diff
@@ -9,14 +9,14 @@
 from cgmodels.cg.constants import Pipeline
 from cg.exc import CgError
 from cg.meta.rsync import RsyncAPI
-from cg.models.cg_config import CGConfig
```

Comment:
I would double check this deletion as it is another module

Reply:
It was unused - so no worries :)